index.html

```diff
@@ -1127,6 +1163,150 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
+                        <p><a href="#writer-applications">Writer Applications</a> perform <a href="https://solidproject.org/TR/protocol#writing-resources">write operations</a> against Solid Protocol <a href="https://solidproject.org/TR/protocol#Server">servers</a>.</p>
+        
+                        <p>Ownership by WebID URI entails that the social agent has authority over the representation of the resource it identifies, including access to perform write operations targeting the Solid WebID Profile.</p>
+                      </div>
+        
+                        <p about="" id="writer-application-create-extended-profile" rel="spec:requirement"
+                            resource="writer-application-create-extended-profile">
+                            <span property="spec:statement"><a rel="spec:requirementSubject" href="#WriterApplication">Writer
+                                    Application</a> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> use an
+                            <code>HTTP PUT</code> or <code>PATCH</code> request targeting the Extended Profile to be created.</span>
+                        </p>
+                        <p about="" id="writer-application-update-extended-profile" rel="spec:requirement"
+                            resource="writer-application-update-extended-profile">
+                            <span property="spec:statement"><a rel="spec:requirementSubject" href="#WriterApplication">Writer
+                                    Application</a> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> use an
+                             <code>HTTP PUT</code> or <code>PATCH</code> request targeting the Solid WebID Profile to be updated</span>
+                        </p>
+                        <p about="" id="writer-application-link-extended-profile-webid" rel="spec:requirement"
+                            resource="writer-application-link-extended-profile-webid">
+                            <span property="spec:statement"><a rel="spec:requirementSubject" href="#WriterApplication">Writer
+                                    Application</a> <span rel="spec:requirementLevel" resource="spec:MUST">MUST</span> update the <a
+                                        href="#solid-profile">Solid WebID Profile </a>by adding a triple of the form
+                                    <code>?webid rdfs:seeAlso ?extendedProfile.</code></span>
+                        </p>
+                        <div class="note" id="sameness-between-identities" inlist="" rel="schema:hasPart"
+                            resource="#sameness-between-identities">
+                            <h5 property="schema:name"><span>Note</span>:</h5>
+                            <div datatype="rdf:HTML" property="schema:description">
+                                <p>The use of <code>owl:sameAs</code> is encouraged when <a href="#use-cases">sameness between identities</a> is
```

Comment:
Given `?webid rdfs:seeAlso ?extendedProfile`, I see:

* `webid` denotes a non-informational resource
* `extendedProfile` denotes an informational resource

I don't see how `owl:sameAs` could be appropriate in this case.

Reply:
The note is not necessarily implying `webid sameAs extendedProfile` either, but I agree that it can can better indicate / emphasise on the URI that identifies the social agent and not the document regarding `owl:sameAs`.

Not sure if this is good enough but some paraphrasing can help (without going into depths about what sameAs is or how to use it properly - those that need something like that can make that call based on knowledge from elsewhere):

```suggestion
                                <p>The use of <code>owl:sameAs</code> is encouraged when <a href="#use-cases">sameness between identities</a> found in Profile documents is
```

Reply:
Would something like this work:

> While the rdfs:seeAlso property is recommended to establish links to extended profiles or additional information, the
use of the owl:sameAs property is encouraged when connecting two WebIDs that denote the same identity, as shown in the example: `?webid owl:sameAs ?webid.`

Reply:
(Everything is more complicated than it seems at first glance...)

Reply:
I would even suggest adding what an understanding may be between: one entity referring to another vs. both entities referring to each other. This generally ties to the other use cases, e.g., verification.